app/Http/Middleware/HandleInertiaRequests.php

```diff
@@ -39,7 +39,7 @@ public function share(Request $request): array
       'ziggy' => fn() => [
         ...(new Ziggy)->toArray(),
         'location' => $request->url(),
-        'query' => $request->query(),
+        'query' => $request->query()['data'] ?? $request->query(),
```

Comment:
This feels, weird. Why was this added?

Reply:
I've updated this to just return an empty array if `$request->query()['data']` is undefined